hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/evm/Evm46ValidationSuite.java

```diff
@@ -609,6 +613,7 @@ private HapiSpec internalCallToNonExistingMirrorAddressResultsInNoopSuccess() {
                                         resultWith().createdContractIdsCount(0).gasUsed(24972))));
     }
 
+    // Adding refusingEthConversion() due to fee differences and not supported address type
```

Comment:
And again I don't get it.  Could these two questions be simply answered by adding Javadoc over there at `HapiContractCreate.refusingEthConversion` that explains what it does and when you'd want to do it?

(And I see there are multiple others of this comment in this suite and others.  But I won't comment on them; just consider all of them included here.)

Reply:
I started adding all the reasons we might want to refuse eth conversion above the `HapiContractCreate.refusingEthConversion` but realized that we have quite a few. I think if we take this approach it would be confusing to know exactly why a certain method refuses a conversion. That's why I just moved the comments near the usage of `refusingEthConversion` to be clearer.